### PR TITLE
Update copyright year to 2021.

### PR DESCRIPTION
**Pull request by rhornung67**
# Summary

- This PR updates the latest copyright year in all files from 2020 to 2021. Woohoo!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 
 [comment]: # (#################################################################)
-[comment]: # (Copyright 2016-19, Lawrence Livermore National Security, LLC)
+[comment]: # (Copyright 2016-20, Lawrence Livermore National Security, LLC)
```
Comment (davidbeckingsale) [inline README.md:3]: This should be 21?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 
 [comment]: # (#################################################################)
-[comment]: # (Copyright 2016-19, Lawrence Livermore National Security, LLC)
+[comment]: # (Copyright 2016-2021, Lawrence Livermore National Security, LLC)
```
Comment (davidbeckingsale) [inline CONTRIBUTING.md:3]: Should this just be `21`?

**Comment by davidbeckingsale** [comment]
Is there a reason to prefer `2016-2021` vs. `2016-21`?

**Comment by rhornung67** [comment]
> Is there a reason to prefer `2016-2021` vs. `2016-21`?

I think we use 2016-21 in all files except the LICENSE file. It needs to be 2016-2021 in that file for GitHub to pick it up and link the license to the main page.

File: CONTRIBUTING.md (hunk 2 above)
Comment (rhornung67) [reply CONTRIBUTING.md]: Fixed. Good catch.

**Comment by davidbeckingsale** [comment]
> > Is there a reason to prefer `2016-2021` vs. `2016-21`?
> 
> I think we use 2016-21 in all files except the LICENSE file. It needs to be 2016-2021 in that file for GitHub to pick it up and link the license to the main page.

Sounds good to me.

File: README.md (hunk 1 above)
Comment (davidbeckingsale) [reply README.md:3]: I'll approve if you fix this one.

Comment (rhornung67) [reply README.md]: Done.

**Review by davidbeckingsale — APPROVED** [review]

**Pull request merged**